### DISCUSSION TITLE
Laser GRBL Help Needed

### DISCUSSION THREAD

**Discussion by LDHTHOPI** [arkypita/LaserGRBL#2223] [General]
New to laser engraving and I'm not IT or anything close.  I have an engraver I bought on Amazon, an LDHTHOPI I think, (it's my username).  It was working great until about three weeks ago, now it wont finish projects.  I keep getting red "abort during cycle" and purple "sys.aborts" messages in the area with the commands and a pop-up on the screen saying problem with the board due to "Machine Alarm." 

I uninstalled and reinstalled the Laser GRBL software, played around with the configuration a little and nothing seems to fix the issue.   

There are two kill switches, X and Y, but the homing function does not work.  When I press the homing button, the motor locks, wont move and makes a light humming noise.  Since homing does not work, I cannot restart where the project crashed. 

Any help would be great.  Thanks!

**Comment by LDHTHOPI** [comment]
Here is a picture of the error message:
![Screenshot (37)](https://github.com/arkypita/LaserGRBL/assets/163628131/131980bc-dcde-4ec7-8cee-330da283c0e3)

**Reply by arkypita** [reply]
It is a machine problem, and it has nothing to do with LaserGRBL.
I don't know what the cause could be, you have to contact the manufacturer.
